test/integration/features/basic.feature

```diff
@@ -10,6 +10,42 @@ Feature: Basic
      Then Minishift should have state "Running"
       And Minishift should have a valid IP address
 
+  Scenario: OpenShift is ready after startup
+    After startup of Minishift OpenShift instance should respond correctly on its html endpoints
+    and OpenShift web console should be accessible.
+    Given Minishift has state "Running"
+     Then body of HTTP request to "OpenShift" at "/healthz" contains "ok"
+      And body of HTTP request to "OpenShift" at "/healthz/ready" contains "ok"
+      And body of HTTP request to "OpenShift" at "/console" contains "<title>OpenShift Web Console</title>"
+
+  Scenario Outline: User can set, get, view and unset values in configuration file
+    User is able to setup persistent configuration of Minishift using "config" command
+    and its subcommands, changing values stored in "config/config.json".
+    #To get values of nested keys, use following dot formating: key.nestedKey.
+    #If an array is expected, use following formating: [value1 value2 value3].
+    Given Minishift has state "Running"
+     When executing "minishift config set <property> <value>" succeeds
+     Then JSON config file "config/config.json" contains key "<property>" with value "<value>"
+      And stdout of command "minishift config get <property>" is equal to "<value>"
+      And stdout of command "minishift config view --format {{.ConfigKey}}:{{.ConfigValue}}" contains "<property>:<value>"
+     When executing "minishift config unset <property>" succeeds
+     Then stdout of command "minishift config get <property>" is equal to "<nil>"
+      And JSON config file "config/config.json" does not contain key "<property>"
+      
+  Examples: Config values to work with
+    | property  | value |
+    | disk-size | 22g   |
+    | memory    | 2222  |
+    | cpus      | 3     |
+
+  Scenario: Commands console and ip returns valid ip
+    User is able to get ip of Minishift VM with command "ip" and url adress of OpenShift instance
+    by running command "console" with flag "--url". Both commands should contain same IP.
+    Given Minishift has state "Running"
+     When executing "minishift console --url" succeeds
+     Then stdout should be valid URL
+      And "minishift ip" and "minishift console --url" should return same IP address
```

Comment:
I find this misleading "return same IP address", 'minishift console --url' does not return an IP, it returns an URL. Also it seems arbitrary to compare these two in a step. Here is what I would suggest:

    Given Minishift has state "Running"
    When executing "minishift console --url" succeeds
    Then stdout should be valid URL
    And stdout should be the OpenShift Console URL 
    When executing  "minishift ip" succeeds 
    Then stdout should be valid IP
    And stdout should be the Minishift VM IP

Not using variables, I think that would be the most concise test. 

Also have you seen https://github.com/DATA-DOG/godog/issues/81#issuecomment-300853730